sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/main/scala/com/azure/cosmos/spark/BulkWriter.scala

```diff
@@ -39,6 +42,8 @@ class BulkWriter(container: CosmosAsyncContainer,
   private val activeTasks = new AtomicInteger(0)
   private val errorCaptureFirstException = new AtomicReference[Throwable]()
   private val bulkInputEmitter: Sinks.Many[CosmosItemOperation] = Sinks.many().unicast().onBackpressureBuffer()
+  private val emitFailureHandler: EmitFailureHandler =
```

Comment:
can be static - meaning on the companion object BulkWriter

Reply:
resolved comments